gpkit/__init__.py

```diff
@@ -90,11 +90,8 @@ def disable_signomials():
 from .variables import Variable, VectorVariable, ArrayVariable
 from .varkey import VarKey
 from .posyarray import PosyArray
-from .geometric_program import GeometricProgram
-from .signomial_program import SignomialProgram
```

Comment:
I appreciate wanting to force designers to create Models.

But I wonder if we might get "mathematician" users who want to create and solve GPs directly, in their mathematical form. That would be the case for exposing `GeometricProgram`.